UITests/UITests.swift

```diff
@@ -11,138 +11,141 @@ import XCTest
 class UITests: UITestCase {
     func testRefreshControl() {
         app.staticTexts["Manage Roster"].tap()
-
+        
         let firstCell = app.staticTexts["Adrienne"]
         let coordinate = firstCell.coordinate(withNormalizedOffset: CGVector(dx: 0, dy: 0))
         let bottom = firstCell.coordinate(withNormalizedOffset: CGVector(dx: 0, dy: 10))
         coordinate.press(forDuration: 0, thenDragTo: bottom)
-
+        
         app.buttons["Dismiss"].tap()
     }
-
+    
     func testElementExists() {
         XCTAssert(app.staticTexts["Volley"].exists)
     }
-
+    
     func testElementExistsWithPredicate() {
         let predicate = NSPredicate(format: "label BEGINSWITH[cd] 'set your team details'")
         let label = app.staticTexts.element(matching: predicate)
         XCTAssert(label.exists)
     }
-
+    
     func testElementWithEllipseExists() {
         app.staticTexts["Manage Roster"].tap()
+        
+        let searchText = "Adolph Blaine Charles David Earl Frederick Gerald Hubert Irvin John Kenneth Lloyd Martin Nero Oliver Paul Quincy Randolph Sherman Thomas Uncas Victor William Xerxes Yancy Wolfeschlegelsteinhausenbergerdorff, Senior"
+        let predicate = NSPredicate(format: "label CONTAINS[c] %@", searchText)
+        let elementQuery = app.staticTexts.containing(predicate)
 
-        let longNameCell = app.staticTexts["Adolph Blaine Charles David Earl Frederick Gerald Hubert Irvin John Kenneth Lloyd Martin Nero Oliver Paul Quincy Randolph Sherman Thomas Uncas Victor William Xerxes Yancy Wolfeschlegelsteinhausenbergerdorff, Senior"]
-        XCTAssert(longNameCell.exists)
+        XCTAssert(elementQuery.count > 0)
```

Comment:
Why doesn't `exist` work here?

Reply:
`exist` works only for shorter comparison of string apparently. There's a similar question on Stackoverflow. Not applying this change fails the test.